.tmuxinator.yml

```diff
@@ -10,6 +10,8 @@ pre_window:
   - alias gateway-cli="\$FM_GATEWAY_CLI"
   - alias mint_rpc_client="\$FM_MINT_RPC_CLIENT"
   - alias fedimint-dbdump="\$FM_DB_DUMP"
+  - alias restart="./scripts/restart-tmux.sh"
+  - alias dbtool="\$FM_DB_TOOL"
```

Comment:
This line seems unrelated? Did it get added by a rebase gone wrong?

Reply:
I accidentally deleted the dbtool line early on, and added it back in with a commit that I squashed in the rebase. I thought I had rebased properly, but I could have messed something up, I'm kind of new to maintaining a tidy history... Thanks for approval! :)